MantidPlot/src/Mantid/MantidGroupPlotGenerator.cpp

```diff
@@ -25,70 +26,97 @@ MantidGroupPlotGenerator::MantidGroupPlotGenerator(MantidDisplayBase *mantidUI)
 
 /**
  * Plots a surface graph from the given workspace group
- * @param wsGroup :: [input] Workspace group to plot
- * @param options :: [input] User-selected plot options
+ * @param accepted :: [input] true if plot has been accepted
+ * @param plotIndex :: [input] plot index
+ * @param axisName :: [input] axis name
+ * @param logName :: [input] log name
+ * @param customLogValues :: [input] custom log values
+ * @param workspaces :: [input] set of workspaces forming the group to be
+ * plotted
  */
 void MantidGroupPlotGenerator::plotSurface(
-    const WorkspaceGroup_const_sptr &wsGroup,
-    const MantidSurfacePlotDialog::UserInputSurface &options) const {
-  plot(Type::Surface, wsGroup, options);
+    bool accepted, int plotIndex, const QString &axisName,
```

Comment:
Renamed MantidSurfaceContourPlotGenerator as implied by opening comment in .h file.